### PR TITLE
tests for tags as refs

### PR DESCRIPTION
**Pull request by andrew-wm-arthur**

### CODE REVIEW

File: cypress/integration/productionTests/publicPaths/render/repository/docsNoTables.spec.ts [hunk 1]
```diff
@@ -9,13 +9,14 @@ import {
...
   testTablesSection,
+  testTagSection,
   testViewsSection,
 } from "../../../../utils/sharedTests/repoLeftNav";
 import { testSqlConsole } from "../../../../utils/sharedTests/sqlEditor";
 
 const pageName = "Repository page with docs and no tables";
 const currentOwner = "automated_testing";
-const currentRepo = "repo_docs_no_tables";
+const currentRepo = "repo_with_tags_and_branches";
```
Comment (tbantle22) [inline cypress/integration/productionTests/publicPaths/render/repository/docsNoTables.spec.ts:19]: Did you mean to change this?

File: cypress/integration/productionTests/publicPaths/render/repository/tags.spec.ts [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  testRepoHeaderWithBranch,
+  testTablesSection,
+  testTagSection,
+  testViewsSection,
+} from "../../../../utils/sharedTests/repoLeftNav";
+import { testSqlConsole } from "../../../../utils/sharedTests/sqlEditor";
+
+const pageName = "Repository page with tables and docs";
+const currentOwner = "automated_testing";
+const currentRepo = "repo_with_tags_and_branches";
```
Comment (tbantle22) [inline cypress/integration/productionTests/publicPaths/render/repository/tags.spec.ts:18]: Can you make this repo on dev too please?

File: cypress/integration/utils/sharedTests/repoLeftNav.ts [hunk 3]
```diff
@@ -427,4 +450,9 @@ export const testSectionsNotVisible: Tests = [
     "[data-cy=repo-query-catalog]",
     notBeVisible,
   ),
+  newExpectation(
+    "should not have repo Tag List section for repo with no data",
+    "[data-cy=repo-tags]",
+    notBeVisible,
+  ),
```
Comment (tbantle22) [inline cypress/integration/utils/sharedTests/repoLeftNav.ts:457]: Same as this comment https://github.com/liquidata-inc/ld/pull/5018#discussion_r485977322

**Review by tbantle22 — APPROVED** [review]

File: cypress/integration/productionTests/publicPaths/render/repository/tags.spec.ts (hunk 2 above)
Comment (andrew-wm-arthur) [reply cypress/integration/productionTests/publicPaths/render/repository/tags.spec.ts:18]: Yes, I'll get perms tomorrow

File: cypress/integration/productionTests/publicPaths/render/repository/docsNoTables.spec.ts (hunk 1 above)
Comment (andrew-wm-arthur) [reply cypress/integration/productionTests/publicPaths/render/repository/docsNoTables.spec.ts:19]: nope, my mistake

**Pull request merged**